roles/loadbalancer/provision/tasks/main.yml

```diff
@@ -0,0 +1,74 @@
+- name: switch selinux to permissive
+  selinux: policy=targeted state=permissive
+
+#this is required to have the 22 port free on haproxy VIP's
+- name: setup ssh for loadbalancer
+  lineinfile: dest=/etc/ssh/sshd_config line="ListenAddress {{ installer.overcloud.loadbalancer.network.plain.ip | ipaddr('address') }}"
+
+- name: restart sshd service
+  service: name=sshd state=restarted
+
+- name: enable vlan kernel module
+  copy: src=vlan.modules
+        dest=/etc/sysconfig/modules/vlan.modules
```

Comment:
`vlan.modules` is a single line. let's use it explicitly instead of storing the file in repo.